test/pdo_sqlsrv/pdo_prepare_invalid_cursor.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+Test PDO::prepare by passing in invalid cursor value
+--SKIPIF--
+
+--FILE--
+<?php
+  
+require_once("autonomous_setup.php");
+
+try 
+{   
+    $database = "tempdb";
+    $conn = new PDO( "sqlsrv:Server = $serverName; Database = $database", $username, $password); 
+    //$conn->setAttribute( PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION );
+    
```

Comment:
what if you set error mode to be ERRMODE_EXCEPTION? Anyway, if you're not planning to use this, please delete the line. :)